libmbp/devices/xiaomi.cpp

```diff
@@ -110,6 +110,19 @@ void addXiaomiDevices(std::vector<Device *> *devices)
     });
     devices->push_back(device);
 
+    // Xiaomi Redmi Note 3 (Snapdragon)
+    device = new Device();
+    device->setId("kenzo");
+    device->setCodenames({ "kenzo" });
+    device->setName("Xiaomi Redmi Note 3 (Snapdragon)");
+    device->setBlockDevBaseDirs({ BOOTDEVICE_BASE_DIR });
```

Comment:
kenzo uses "/dev/block/platform/soc.0/7824900.sdhci/by-name" in mm and "/dev/block/platform/7824900.sdhci/by-name" in lp i tried compiling it last week and i faced the same issues as discussed here https://github.com/chenxiaolong/DualBootPatcher/pull/40